### PR TITLE
feat: Added Israel location data (#459)

### PR DESCRIPTION
**Pull request by zoearose**
* Update location.cpp
* Update location_data.h
* Update location_test.cpp
* Run format_code script

test: location_test.cpp was updated
fix: typos & formatting issues due to the language being read Right to Left
fix: location_test.cpp

* Update location_test.cpp
* Add Israel data to switch case

### CODE REVIEW

File: CMakeCache.txt [hunk 1]
```diff
@@ -0,0 +1,359 @@
+# This is the CMakeCache file.
```
Comment (cieslarmichal) [inline CMakeCache.txt:1]: I dont think those files should be commited to repo, could you add them to gitignore?

File: Testing/Temporary/CTestCostData.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+---
```
Comment (cieslarmichal) [inline Testing/Temporary/CTestCostData.txt:1]: please remove this file from your changes

File: Testing/Temporary/LastTest.log [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Start testing: Oct 31 12:06 EDT
```
Comment (cieslarmichal) [inline Testing/Temporary/LastTest.log:1]: please remove this file from your changes

File: tests/specific_test_log.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+generated building number: רחו
```
Comment (cieslarmichal) [inline tests/specific_test_log.txt:1]: dont commit this file

File: CMakeCache.txt (hunk 1 above)
Comment (zoearose) [reply CMakeCache.txt]: Sounds good! I made all the changes you requested, let me know if you need anything else!

**Review by cieslarmichal — APPROVED** [review]

**Pull request merged**